dvc/command/add.py

```diff
@@ -14,38 +16,44 @@ class CmdAdd(CmdBase):
     def run(self):
         try:
             if len(self.args.targets) > 1 and self.args.file:
-                raise DvcException("can't use '--file' with multiple targets")
-
-            for target in self.args.targets:
-                self.repo.add(
-                    target,
-                    recursive=self.args.recursive,
-                    no_commit=self.args.no_commit,
-                    fname=self.args.file,
+                raise RecursiveAddingWhileUsingFilename(
+                    "can't use '--file' with multiple targets"
                 )
 
-        except DvcException:
-            logger.exception("failed to add file")
+            with Tqdm(
```

Comment:
It would be really nice to see before and after comparison, something like you've done with asciinema before. It really helps visualize this stuff.